### PR TITLE
FOGL-6696 Add more tuning sections to the documentation

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/tuning_fledge.rst [hunk 1]
```diff
@@ -39,7 +42,7 @@ The south services within Fledge each have a set of advanced configuration optio
 
   - *Reading Rate Per* - This defines the units to be used in the *Reading Rate* value. It allows the selection of per *second*, *minute* or *hour*.
 
-  - *Minimum Log Level* - This configuration option can be used to set the logs that will be seen for this service. It defines the level of logging that is send to the syslog and may be set to *error*, *warning*, *info* or *debug*. Logs of the level selected and higher will be sent to the syslog.
+  - *Minimum Log Level* - This configuration option can be used to set the logs that will be seen for this service. It defines the level of logging that is send to the syslog and may be set to *error*, *warning*, *info* or *debug*. Logs of the level selected and higher will be sent to the syslog. You may access the contents of these logs by selecting the log icon in the bottom right of this screen.
```
Comment (ashish-jabble) [inline docs/tuning_fledge.rst:45]: bottom left

File: docs/tuning_fledge.rst [hunk 2]
```diff
@@ -49,3 +52,50 @@ The tuning of the south service allows the way the buffering is used within the
...
+North Advanced Configuration
+============================
+
+In a similar way to the south services, north services and tasks also have advanced configuration that can be used to tune the operation of the north side of Fledge. The north advanced configuration is accessed in much the same way as the south, select the North page and open the particular north service or task. A tabbed screen will be shown which contains an *Advanced Configuration* tab.
+
++------------------+
+| |north_advanced| |
++------------------+
+
+  - *Minimum Log Level* - This configuration option can be used to set the logs that will be seen for this service or task. It defines the level of logging that is send to the syslog and may be set to *error*, *warning*, *info* or *debug*. Logs of the level selected and higher will be sent to the syslog. You may access the contents of these logs by selecting the log icon in the bottom right of this screen.
```
Comment (ashish-jabble) [inline docs/tuning_fledge.rst:65]: bottom left

File: docs/tuning_fledge.rst [hunk 3]
```diff
@@ -49,3 +52,50 @@ The tuning of the south service allows the way the buffering is used within the
...
+Scheduler
+=========
+
+The Fledge core contains a scheduler that is used for running periodic tasks, this scheduler has a couple of tuning parameters. To access these parameters from the Fledge User Interface, in the configuration page select the *Advanced* options and then the *Scheduler* section.
+
++----------------------+
+| |scheduler_advanced| |
++----------------------+
+
+  - *Max Running Tasks* - Specifies the maximum number of tasks that can be running at any one time. This parameter is designed to stop runaway tasks adversely impacting the performance of the system. When this number is reached no new tasks will be created until one or more of the currently running tasks terminated. Set this two low and you will not be able to run all the task you require in parallel. Set it too high and the system is more at risk from runaway tasks.
```
Comment (ashish-jabble) [inline docs/tuning_fledge.rst:95]: to/too? low

File: docs/tuning_fledge.rst (hunk 1 above)
Comment (MarkRiddoch) [reply docs/tuning_fledge.rst]: I really should know my left form my right!

File: docs/tuning_fledge.rst (hunk 3 above)
Comment (MarkRiddoch) [reply docs/tuning_fledge.rst]: Well spotted

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**